cli_interactive.py

```diff
@@ -20,13 +20,17 @@
 #([1,4,3], 'ord', True, 'C')
 def main():
 	entered_values, binary_order, show_binary, initial_pitch = prompt_for_all_arguments()
-	common4.run_standard_text_flow(get_intervals, length_needed, total_num_pitches, make_permutations, pitch_list_generator, pitch_stripper, tallies, generate_unique_name, later_function, footer)
+	common4.run_standard_text_flow(entered_values, binary_order, show_binary, initial_pitch)
 #(interval_set, seq_length, total, perm_list, pitches_list_all, string_list, total_tally, filename, decision, footer)
 
 
 def prompt_for_all_arguments():# -> Tuple[List[int], str, bool, str]:
     print("Enter sequence of intervals as number of half steps \n1=minor 2nd, 2=Major 2nd, 3=minor 3rd, \n4=Major 3rd, 5=Perfect 4th, 6=Tri-tone")
-    entered_values = input("Enter a sequence of intervals separated by a comma: \nonly valid intervals: 1 - 6\n >")
+    entered_values_raw = input("Enter a sequence of intervals separated by a comma: \nonly valid intervals: 1 - 6\n >")
+    entered_values = [int(x.strip()) for x in entered_values_raw.split(',')]
```

Comment:
you can tell in my type int above on line 27 that this is supposed to be a List of integers, not a str.

without this change, we get

```
Traceback (most recent call last):
  File "cli_interactive.py", line 44, in <module>
    main()
  File "cli_interactive.py", line 23, in main
    common4.run_standard_text_flow(entered_values, binary_order, show_binary, initial_pitch)
  File "/Users/jon/repos/music-intervals/common4.py", line 13, in run_standard_text_flow
    interval_set = get_intervals(entered_values)
  File "/Users/jon/repos/music-intervals/formatting5.py", line 28, in get_intervals
    return [intervals[val] for val in entered_values]
  File "/Users/jon/repos/music-intervals/formatting5.py", line 28, in <listcomp>
    return [intervals[val] for val in entered_values]
KeyError: '1'
```

and error within `get_intervals`